sdk/search/azure-search-documents/azure/search/documents/_internal/_search_indexing_buffered_sender.py

```diff
@@ -173,54 +180,70 @@ def _reset_timer(self):
             self._timer.cancel()
         except AttributeError:
             pass
-        self._timer = threading.Timer(self._window, self._process)
+        self._timer = threading.Timer(self._auto_flush_interval, self._process)
         if self._auto_flush:
             self._timer.start()
 
-    def add_upload_actions(self, documents):
+    @distributed_trace
+    def upload_documents(self, documents, **kwargs):  # pylint: disable=unused-argument
         # type: (List[dict]) -> None
         """Queue upload documents actions.
 
         :param documents: A list of documents to upload.
         :type documents: List[dict]
         """
         actions = self._index_documents_batch.add_upload_actions(documents)
-        self._new_callback(actions)
+        self._new_action_callback(actions)
```

Comment:
nit: i think it would be more clear if the private variables tracking the `on_` hooks were named the same as what users would pass in (in this case, self._on_new)